doc/build_apps/release_policy.rst

```diff
@@ -81,8 +81,13 @@ REST API guarantees spelled out in the first section apply across releases, but
     :alt: Release
     
 CCF 1.0 Sunset
------------------
+--------------
```

Comment:
I think we should write these in a table rather than separate verbose sections so that we don't use tenses that never go out of date (e.g. "**will go** out of support on June 1, 2023" will sound odd after that date). 

+---------------+---------------------+----------------------+
| Major release  | First release date    | Out of support date |
+============+=========+=============+
| 1.0                   | April 30, 2021 | June 30, 2022           |
+---------------+----------------------+---------------------+
| 2.0                   | May 17, 2022          | June 1, 2023            |
+---------------+----------------------+---------------------+
| 3.0                   | November 24, 2022 | TBC                         |
+---------------+----------------------+---------------------+
| 4.0                   | Upcoming               | TBD                         |
+----------------+---------------------+---------------------+